packages/grafana-runtime/src/services/pluginExtensions/getPluginExtensions.ts

```diff
@@ -1,13 +1,15 @@
-import { PluginExtension } from '@grafana/data';
+import { type PluginExtension, type PluginExtensionLink, type PluginExtensionComponent } from '@grafana/data';
```

Comment:
nit: can also put the type keyword after import.

```suggestion
import type { PluginExtension, PluginExtensionLink, PluginExtensionComponent } from '@grafana/data';
```

Reply:
Makes sense 👍

Reply:
Fixed in https://github.com/grafana/grafana/pull/68600/commits/ce1a2d78f027f95189114ea8314b2ea9fde51b43.